aiidalab/app.py

```diff
@@ -203,15 +214,13 @@ def available_versions(
     ) -> Generator[str, None, None]:
         """Return a list of available versions excluding the ones with core dependency conflicts."""
         if self.is_registered():
-            for version in sorted(self.releases, key=parse, reverse=True):
+            for version in sort_semantic(self.releases, prereleases=prereleases):
                 version_requirements = self.parse_python_requirements(
                     self.releases[version]
                     .get("environment", {})
                     .get("python_requirements", [])
                 )
-                if (
-                    prereleases or not parse(version).is_prerelease
```

Comment:
This part of the if statement is not longer needed since pre-releases are filtered out in `sort_semantic` already.